privaterelay/models.py

```diff
@@ -326,6 +326,16 @@ def has_vpn(self) -> bool:
                 return True
         return False
 
+    @property
+    def has_megabundle(self) -> bool:
+        if not self.fxa:
+            return False
+        user_subscriptions = self.fxa.extra_data.get("subscriptions", [])
+        for sub in settings.SUBSCRIPTIONS_WITH_MEGABUNDLE:
+            if sub in user_subscriptions:
+                return True
```

Comment:
suggestion (blocking): This logic says if the user has ANY of these subscriptions: `guardian_vpn_1,guardian_vpn,monitor`, they have the megabundle. It needs to say if the user as ALL of these subscriptions.

```suggestion
        if all(sub in user_subscriptions for sub in settings.SUBSCRIPTIONS_WITH_MEGABUNDLE):
            return True
```